Crash on world load in newest version in development (MC 1.18.2, FREX v6.0.249)

I use JMX in my mod and through it FREX. If I use JMX 1.22.248 (so FREX v6.0.249) it crashes after it has finished loading the world and starts rendering. If I go back one version (JMX 1.22.247 and FREX v6.0.248), there are no issues.

Crash report below.

[crash-2022-04-18_13.51.20-client.txt](https://github.com/vram-guild/frex/files/8505470/crash-2022-04-18_13.51.20-client.txt)
.

Additional logs:
```
[13:58:04] [Render thread/ERROR] (FabricLoader/Mixin) Mixin apply for mod frex failed frex.pastel.json:MixinChunkRebuildTask -> net.minecraft.client.render.chunk.ChunkBuilder$BuiltChunk$RebuildTask: org.spongepowered.asm.mixin.transformer.throwables.InvalidMixinException @Shadow field this$1 was not located in the target class net.minecraft.client.render.chunk.ChunkBuilder$BuiltChunk$RebuildTask. Using refmap frex-refmap.json
 org.spongepowered.asm.mixin.transformer.throwables.InvalidMixinException: @Shadow field this$1 was not located in the target class net.minecraft.client.render.chunk.ChunkBuilder$BuiltChunk$RebuildTask. Using refmap frex-refmap.json
	at org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.attachFields(MixinPreProcessorStandard.java:624) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.attach(MixinPreProcessorStandard.java:302) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.createContextFor(MixinPreProcessorStandard.java:277) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinInfo.createContextFor(MixinInfo.java:1289) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:292) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.apply(TargetClassContext.java:421) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:403) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:247) ~[fabric-loader-0.13.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:150) ~[fabric-loader-0.13.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:155) ~[fabric-loader-0.13.3.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at net.minecraft.client.render.BuiltChunkStorage.createChunks(BuiltChunkStorage.java:45) ~[minecraft-project-@-merged-named.jar:?]
	at net.minecraft.client.render.BuiltChunkStorage.<init>(BuiltChunkStorage.java:30) ~[minecraft-project-@-merged-named.jar:?]
	at net.minecraft.client.render.WorldRenderer.reload(WorldRenderer.java:692) ~[minecraft-project-@-merged-named.jar:?]
	at net.minecraft.client.render.WorldRenderer.setWorld(WorldRenderer.java:641) ~[minecraft-project-@-merged-named.jar:?]
	at net.minecraft.client.MinecraftClient.setWorld(MinecraftClient.java:1978) ~[minecraft-project-@-merged-named.jar:?]
	at net.minecraft.client.MinecraftClient.joinWorld(MinecraftClient.java:1911) ~[minecraft-project-@-merged-named.jar:?]
	at net.minecraft.client.network.ClientPlayNetworkHandler.onGameJoin(ClientPlayNetworkHandler.java:364) ~[minecraft-project-@-merged-named.jar:?]
	at net.minecraft.network.packet.s2c.play.GameJoinS2CPacket.apply(GameJoinS2CPacket.java:48) ~[minecraft-project-@-merged-named.jar:?]
	at net.minecraft.network.packet.s2c.play.GameJoinS2CPacket.apply(GameJoinS2CPacket.java:23) ~[minecraft-project-@-merged-named.jar:?]
	at net.minecraft.network.NetworkThreadUtils.method_11072(NetworkThreadUtils.java:26) ~[minecraft-project-@-merged-named.jar:?]
	at net.minecraft.util.thread.ThreadExecutor.executeTask(ThreadExecutor.java:150) [minecraft-project-@-merged-named.jar:?]
	at net.minecraft.util.thread.ReentrantThreadExecutor.executeTask(ReentrantThreadExecutor.java:29) [minecraft-project-@-merged-named.jar:?]
	at net.minecraft.util.thread.ThreadExecutor.runTask(ThreadExecutor.java:126) [minecraft-project-@-merged-named.jar:?]
	at net.minecraft.util.thread.ThreadExecutor.runTasks(ThreadExecutor.java:120) [minecraft-project-@-merged-named.jar:?]
	at net.minecraft.client.MinecraftClient.render(MinecraftClient.java:1030) [minecraft-project-@-merged-named.jar:?]
	at net.minecraft.client.MinecraftClient.run(MinecraftClient.java:724) [minecraft-project-@-merged-named.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:213) [minecraft-project-@-merged-named.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:416) [fabric-loader-0.13.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77) [fabric-loader-0.13.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.13.3.jar:?]
	at net.fabricmc.devlaunchinjector.Main.main(Main.java:86) [dev-launch-injector-0.2.1+build.8.jar:?]
[13:58:04] [Render thread/ERROR] (Minecraft) Error executing task on Client
 java.lang.RuntimeException: Mixin transformation of net.minecraft.client.render.chunk.ChunkBuilder$BuiltChunk$RebuildTask failed
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:252) ~[fabric-loader-0.13.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:150) ~[fabric-loader-0.13.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:155) ~[fabric-loader-0.13.3.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at net.minecraft.client.render.BuiltChunkStorage.createChunks(BuiltChunkStorage.java:45) ~[minecraft-project-@-merged-named.jar:?]
	at net.minecraft.client.render.BuiltChunkStorage.<init>(BuiltChunkStorage.java:30) ~[minecraft-project-@-merged-named.jar:?]
	at net.minecraft.client.render.WorldRenderer.reload(WorldRenderer.java:692) ~[minecraft-project-@-merged-named.jar:?]
	at net.minecraft.client.render.WorldRenderer.setWorld(WorldRenderer.java:641) ~[minecraft-project-@-merged-named.jar:?]
	at net.minecraft.client.MinecraftClient.setWorld(MinecraftClient.java:1978) ~[minecraft-project-@-merged-named.jar:?]
	at net.minecraft.client.MinecraftClient.joinWorld(MinecraftClient.java:1911) ~[minecraft-project-@-merged-named.jar:?]
	at net.minecraft.client.network.ClientPlayNetworkHandler.onGameJoin(ClientPlayNetworkHandler.java:364) ~[minecraft-project-@-merged-named.jar:?]
	at net.minecraft.network.packet.s2c.play.GameJoinS2CPacket.apply(GameJoinS2CPacket.java:48) ~[minecraft-project-@-merged-named.jar:?]
	at net.minecraft.network.packet.s2c.play.GameJoinS2CPacket.apply(GameJoinS2CPacket.java:23) ~[minecraft-project-@-merged-named.jar:?]
	at net.minecraft.network.NetworkThreadUtils.method_11072(NetworkThreadUtils.java:26) ~[minecraft-project-@-merged-named.jar:?]
	at net.minecraft.util.thread.ThreadExecutor.executeTask(ThreadExecutor.java:150) [minecraft-project-@-merged-named.jar:?]
	at net.minecraft.util.thread.ReentrantThreadExecutor.executeTask(ReentrantThreadExecutor.java:29) [minecraft-project-@-merged-named.jar:?]
	at net.minecraft.util.thread.ThreadExecutor.runTask(ThreadExecutor.java:126) [minecraft-project-@-merged-named.jar:?]
	at net.minecraft.util.thread.ThreadExecutor.runTasks(ThreadExecutor.java:120) [minecraft-project-@-merged-named.jar:?]
	at net.minecraft.client.MinecraftClient.render(MinecraftClient.java:1030) [minecraft-project-@-merged-named.jar:?]
	at net.minecraft.client.MinecraftClient.run(MinecraftClient.java:724) [minecraft-project-@-merged-named.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:213) [minecraft-project-@-merged-named.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:416) [fabric-loader-0.13.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77) [fabric-loader-0.13.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.13.3.jar:?]
	at net.fabricmc.devlaunchinjector.Main.main(Main.java:86) [dev-launch-injector-0.2.1+build.8.jar:?]
Caused by: org.spongepowered.asm.mixin.transformer.throwables.MixinTransformerError: An unexpected critical error was encountered
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:392) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:247) ~[fabric-loader-0.13.3.jar:?]
	... 28 more
Caused by: org.spongepowered.asm.mixin.throwables.MixinApplyError: Mixin [frex.pastel.json:MixinChunkRebuildTask] from phase [DEFAULT] in config [frex.pastel.json] from mod [frex] FAILED during APPLY
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.handleMixinError(MixinProcessor.java:638) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.handleMixinApplyError(MixinProcessor.java:589) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:379) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:247) ~[fabric-loader-0.13.3.jar:?]
	... 28 more
Caused by: org.spongepowered.asm.mixin.transformer.throwables.InvalidMixinException: @Shadow field this$1 was not located in the target class net.minecraft.client.render.chunk.ChunkBuilder$BuiltChunk$RebuildTask. Using refmap frex-refmap.json
	at org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.attachFields(MixinPreProcessorStandard.java:624) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.attach(MixinPreProcessorStandard.java:302) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.createContextFor(MixinPreProcessorStandard.java:277) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinInfo.createContextFor(MixinInfo.java:1289) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:292) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.apply(TargetClassContext.java:421) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:403) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:247) ~[fabric-loader-0.13.3.jar:?]
	... 28 more
[13:58:04] [Render thread/ERROR] (Minecraft) Error executing task on Client
 java.lang.NullPointerException: Cannot invoke "net.minecraft.client.network.ClientPlayerEntity.setServerBrand(String)" because "this.client.player" is null
	at net.minecraft.client.network.ClientPlayNetworkHandler.onCustomPayload(ClientPlayNetworkHandler.java:1635) ~[minecraft-project-@-merged-named.jar:?]
	at net.minecraft.network.packet.s2c.play.CustomPayloadS2CPacket.apply(CustomPayloadS2CPacket.java:60) ~[minecraft-project-@-merged-named.jar:?]
	at net.minecraft.network.packet.s2c.play.CustomPayloadS2CPacket.apply(CustomPayloadS2CPacket.java:36) ~[minecraft-project-@-merged-named.jar:?]
	at net.minecraft.network.NetworkThreadUtils.method_11072(NetworkThreadUtils.java:26) ~[minecraft-project-@-merged-named.jar:?]
	at net.minecraft.util.thread.ThreadExecutor.executeTask(ThreadExecutor.java:150) [minecraft-project-@-merged-named.jar:?]
	at net.minecraft.util.thread.ReentrantThreadExecutor.executeTask(ReentrantThreadExecutor.java:29) [minecraft-project-@-merged-named.jar:?]
	at net.minecraft.util.thread.ThreadExecutor.runTask(ThreadExecutor.java:126) [minecraft-project-@-merged-named.jar:?]
	at net.minecraft.util.thread.ThreadExecutor.runTasks(ThreadExecutor.java:120) [minecraft-project-@-merged-named.jar:?]
	at net.minecraft.client.MinecraftClient.render(MinecraftClient.java:1030) [minecraft-project-@-merged-named.jar:?]
	at net.minecraft.client.MinecraftClient.run(MinecraftClient.java:724) [minecraft-project-@-merged-named.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:213) [minecraft-project-@-merged-named.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:416) [fabric-loader-0.13.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77) [fabric-loader-0.13.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.13.3.jar:?]
	at net.fabricmc.devlaunchinjector.Main.main(Main.java:86) [dev-launch-injector-0.2.1+build.8.jar:?]
[13:58:04] [Render thread/ERROR] (Minecraft) Error executing task on Client
 java.lang.NullPointerException: Cannot invoke "net.minecraft.entity.player.PlayerEntity.getAbilities()" because "playerEntity" is null
	at net.minecraft.client.network.ClientPlayNetworkHandler.onPlayerAbilities(ClientPlayNetworkHandler.java:1461) ~[minecraft-project-@-merged-named.jar:?]
	at net.minecraft.network.packet.s2c.play.PlayerAbilitiesS2CPacket.apply(PlayerAbilitiesS2CPacket.java:65) ~[minecraft-project-@-merged-named.jar:?]
	at net.minecraft.network.packet.s2c.play.PlayerAbilitiesS2CPacket.apply(PlayerAbilitiesS2CPacket.java:25) ~[minecraft-project-@-merged-named.jar:?]
	at net.minecraft.network.NetworkThreadUtils.method_11072(NetworkThreadUtils.java:26) ~[minecraft-project-@-merged-named.jar:?]
	at net.minecraft.util.thread.ThreadExecutor.executeTask(ThreadExecutor.java:150) [minecraft-project-@-merged-named.jar:?]
	at net.minecraft.util.thread.ReentrantThreadExecutor.executeTask(ReentrantThreadExecutor.java:29) [minecraft-project-@-merged-named.jar:?]
	at net.minecraft.util.thread.ThreadExecutor.runTask(ThreadExecutor.java:126) [minecraft-project-@-merged-named.jar:?]
	at net.minecraft.util.thread.ThreadExecutor.runTasks(ThreadExecutor.java:120) [minecraft-project-@-merged-named.jar:?]
	at net.minecraft.client.MinecraftClient.render(MinecraftClient.java:1030) [minecraft-project-@-merged-named.jar:?]
	at net.minecraft.client.MinecraftClient.run(MinecraftClient.java:724) [minecraft-project-@-merged-named.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:213) [minecraft-project-@-merged-named.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
Caused by: org.spongepowered.asm.mixin.transformer.throwables.MixinTransformerError: An unexpected critical error was encountered

	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:416) [fabric-loader-0.13.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77) [fabric-loader-0.13.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.13.3.jar:?]
	at net.fabricmc.devlaunchinjector.Main.main(Main.java:86) [dev-launch-injector-0.2.1+build.8.jar:?]
[13:58:04] [Render thread/ERROR] (Minecraft) Error executing task on Client
Caused by: org.spongepowered.asm.mixin.throwables.MixinApplyError: Mixin [frex.pastel.json:MixinChunkRebuildTask] from phase [DEFAULT] in config [frex.pastel.json] from mod [frex] FAILED during APPLY

 java.lang.NullPointerException: Cannot invoke "net.minecraft.client.network.ClientPlayerEntity.getInventory()" because "this.client.player" is null
	at net.minecraft.client.network.ClientPlayNetworkHandler.onUpdateSelectedSlot(ClientPlayNetworkHandler.java:496) ~[minecraft-project-@-merged-named.jar:?]
	at net.minecraft.network.packet.s2c.play.UpdateSelectedSlotS2CPacket.apply(UpdateSelectedSlotS2CPacket.java:29) ~[minecraft-project-@-merged-named.jar:?]
	at net.minecraft.network.packet.s2c.play.UpdateSelectedSlotS2CPacket.apply(UpdateSelectedSlotS2CPacket.java:15) ~[minecraft-project-@-merged-named.jar:?]
	at net.minecraft.network.NetworkThreadUtils.method_11072(NetworkThreadUtils.java:26) ~[minecraft-project-@-merged-named.jar:?]
	at net.minecraft.util.thread.ThreadExecutor.executeTask(ThreadExecutor.java:150) [minecraft-project-@-merged-named.jar:?]
	at net.minecraft.util.thread.ReentrantThreadExecutor.executeTask(ReentrantThreadExecutor.java:29) [minecraft-project-@-merged-named.jar:?]
	at net.minecraft.util.thread.ThreadExecutor.runTask(ThreadExecutor.java:126) [minecraft-project-@-merged-named.jar:?]
Caused by: org.spongepowered.asm.mixin.transformer.throwables.InvalidMixinException: @Shadow field this$1 was not located in the target class net.minecraft.client.render.chunk.ChunkBuilder$BuiltChunk$RebuildTask. Using refmap frex-refmap.json

	at net.minecraft.util.thread.ThreadExecutor.runTasks(ThreadExecutor.java:120) [minecraft-project-@-merged-named.jar:?]
	at net.minecraft.client.MinecraftClient.render(MinecraftClient.java:1030) [minecraft-project-@-merged-named.jar:?]
	at net.minecraft.client.MinecraftClient.run(MinecraftClient.java:724) [minecraft-project-@-merged-named.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:213) [minecraft-project-@-merged-named.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:416) [fabric-loader-0.13.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77) [fabric-loader-0.13.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.13.3.jar:?]
	at net.fabricmc.devlaunchinjector.Main.main(Main.java:86) [dev-launch-injector-0.2.1+build.8.jar:?]
```

There's a bunch more errors before it crashes but they are all caused by the above it seems. Possibly something recently happened that disabled the fix in #7 because it seems related to that. 

Hopefully this information helps!

--- a/fabriquilt/src/main/java/io/vram/frex/pastel/mixin/MixinChunkRebuildTask.java
+++ b/fabriquilt/src/main/java/io/vram/frex/pastel/mixin/MixinChunkRebuildTask.java
@@ -61,7 +61,7 @@ import io.vram.frex.pastel.mixinterface.RenderChunkRegionExt;
 @Mixin(targets = "net.minecraft.client.renderer.chunk.ChunkRenderDispatcher$RenderChunk$RebuildTask")
 public abstract class MixinChunkRebuildTask implements RenderRegionContext<BlockAndTintGetter> {
 	//e -> field_20839 -> this$1
-	@Shadow protected RenderChunk this$1;
+	@Shadow(aliases = {"field_20839"}) protected RenderChunk this$1;
 
 	// Below are for RenderRegionBakeListener support
 

